nemo/collections/llm/quantization/utils.py

```diff
@@ -73,6 +73,7 @@ def load_with_modelopt_layer_spec(
     nemo_checkpoint_path: str,
     tensor_model_parallel_size: int = 1,
     pipeline_model_parallel_size: int = 1,
+    num_nodes: int = 1,
```

Comment:
I think we also need a separate parameter for devices as I think it refers to num gpus per node